app/models/denounce.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
+#
+# Table name: denounces
+#
+#  id          :bigint           not null, primary key
+#  anonymouse  :boolean          default(FALSE)
+#  area        :integer
+#  description :string
+#  event_date  :date
+#  title       :string
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  user_id     :integer
+#
```

Comment:
If all these comments are not necessary you can remove them, so that the code looks cleaner.